installer/resources/pacbot_app/files/DB.sql

```diff
@@ -771,6 +773,8 @@ CREATE TABLE IF NOT EXISTS `oauth_user_roles` (
   `modifiedDate` datetime DEFAULT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+ALTER TABLE oauth_user_roles ADD PRIMARY KEY (roleId);
```

Comment:
This kind of DDL makes problem to the successive DB import as it tries to add primary key to an already existing key. So you should create a procedure for this. Please follow the logic already I added here 
https://github.com/tmobile/pacbot/blob/beaf950db0dbdd703f8dbc275d619c8e14a4f694/installer/resources/pacbot_app/files/DB.sql#L819